src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/AccediAPiattaformaNotifichePage.java

```diff
@@ -80,22 +74,23 @@ public class AccediAPiattaformaNotifichePage extends BasePage {
     @FindBy(xpath = "//div[@data-testid='pagopa-item']")
     List<WebElement> pagopaItems;
 
+    @FindBy(xpath = "//*[@id=\"root\"]/div[1]/div/main/div/div/div[1]/div[3]/div[4]/div/button[2]")
+    List<WebElement> pagopaAllegatoItems;
+
+
+
     public AccediAPiattaformaNotifichePage(WebDriver driver) {
         super(driver);
     }
 
     public void waitLoadAccediAPiattaformaNotifichePage() {
-        try {
-            By titleLabel = By.id("login-page-title");
-            By loginBy = By.id("login-button");
+
+            By titleLabel = By.id("login-mode-page-title");
+            By loginBy = By.id("spidButton");
```

Comment:
is present 2 file for this login attention for other file

Reply:
Verificato gestione corretta